apps/homescreen/js/app.js

```diff
@@ -523,6 +527,14 @@ const SETTINGS_VERSION = 0;
 
       // Display the top shadow when scrolling down
       case 'scroll':
+        if (e.target === this.panels) {
```

Comment:
Yes, and unfortunately not - not until https://bugzilla.mozilla.org/show_bug.cgi?id=1172171 anyway.